### ISSUE TITLE
Foundation6 <a href> does not work from <li> submenu in top-bar.

### ISSUE THREAD

**Issue by peterstouw** [zurb/foundation-sites#7959]
Hi,
When working on an old website, converting it from a table-driven to a foundation6 website, I discovered that the href in the <a>-tag within a submenu in the top-bar menu (example: page http://petertouw.nl/news-magazine.html) does not work in the web browsers on my computer, it does work in Safari on my iPad2??. In the browser the url is shown but when clicking the <li> nothing happens. I disabled the generated app.css (renaming it to _app.css) and the <li> is clickable and works.
To find out what could cause the problem I did set up a test website (http://www.petertouw.nl/) only using the templates (http://foundation.zurb.com/templates.html) from the foundation website and a few building blocks (http://zurb.com/building-blocks). To check the hrefs I changed all the href="#" to existing urls. This test website I created with Yeti Launch, everything downloaded today (2016-01-21). And all the changes I did make, Yeti Launch was running.
Running MacOSX 10.10.5, YetiLaunch Version 1.0.1 (1.0.1). 
Browsers: Firefox 43.0.4, Chrome Version 47.0.2526.111 (64-bit), Safari Version 9.0.3 (10601.4.4). 
By the way the building block Top Bar With Off-Canvas F6 (http://zurb.com/building-blocks/top-bar-with-off-canvas) works on the foundation web site, clicking [Item 1 > Left Wide 1] in Chrome.
I included the zipped dist folder.
How do I solve this problem?
Thanks in advance.
Peter Touw.
[dist.zip](https://github.com/zurb/foundation-sites/files/99788/dist.zip)

**Comment by zurbchris** [comment]
This is an issue with yeti launch not quite supporting updating foundation with ease. You're on an older version of Foundation 6, and *many* bug fixes and a few features have been added since. If you are comfortable in the command line, you can cd into the directory your project is held and 
```bash
npm i && bower install 
```
If not, I can probably talk you through the process of updating manually, based on the defaults paths set by Yeti Launch.

**Comment by peterstouw** [comment]
Hi Chris,

Thank you for your fast response.

Yesterday I installed a new project with Yeti Launch, I copied the folder, after that I cd into that folder and I ran 'npm i && bower install'.
Result:
Last login: Fri Jan 22 13:20:39 on ttys000
Peters-Mac-Pro:~ petertouw$ cd /Applications/MAMP/htdocs/yetilaunchclean_2016-01-21
Peters-Mac-Pro:yetilaunchclean_2016-01-21 petertouw$ npm i && bower install
Peters-Mac-Pro:yetilaunchclean_2016-01-21 petertouw$

After that I compared the 2 folders, both folders are identical. To me, nothing changed. Compare screenshots included.
I also inserted a submenu topbar into the index.html, still does not work.

I also created another folder from this download foundation-sites-6.1.1.zip. I ran 'npm i && bower install' and npm did a lot.
see file: foundationsites611b - install.rtf.
After that I ran 'npm start', looks ok, did create the document site… and the submenu topbar works.
see file: foundationsites611b - start.rtf.
So the next step, how can I create a Yeti Launch environment, (src/pages etc. and dist folder structure) with these - the right - components. I think I need your help, because I will not solve this on my own quickly.
By the way, I'm not afraid of the command line, when I started in IT there were no GUI's, only command lines.

Thanks,
Peter.
[foundation - a href #2.zip](https://github.com/zurb/foundation-sites/files/100868/foundation.-.a.href.2.zip)

**Comment by zurbchris** [comment]
We didn't want to really put this out there, but you're actually talking to me, so you should probably know. When we initially released Yeti Launch, we were having issues getting a certificate from Apple for the installation process and didn't yet have it configured to auto-update, that issue has been tinkered with, but I'm not actually sure if it's rectified. That being the case, we'll be getting auto-updates working at some point, but in the meantime, you would have to download a new copy of Yeti Launch that comes with the latest version of Foundation.

**Comment by peterstouw** [comment]
Hi Chris,

I installed everything from the command line. All works okay now.
This is what I did:
cd /Applications/MAMP/htdocs
WORKS: sudo npm install --global foundation-cli
FAILS: foundation new foundationsites611dev --framework sites --template zurb
WORKS: foundation new foundationsites611dev
DOWNLOAD: foundation-zurb-template-master.zip
UNZIP: foundation-zurb-template-master.zip to folder foundation-zurb-template-master
FINDER: go into foundationsites611dev folder
FINDER: rename org bower.json to bower[org].json
FINDER: rename org README.md to README[org].md
FINDER: copy the files from the downloaded zurb-template folder into the root of the folder foundationsites611dev
cd /Applications/MAMP/htdocs/foundationsites611dev
WORKS: npm install
WORKS: bower install
WORKS: npm run build (dist folder created)
WORKS: npm start (monitor changes)

Thank you all at Zurb for the nice software.
Peter.
